{
  try {
    internalRun();
  }
 catch (  IOException e) {
    System.out.println("Store processing interrupted " + store + " because of IOException "+ e.getMessage());
    e.printStackTrace();
  }
}
